halo/app/start.go

```diff
@@ -268,3 +273,19 @@ func newEngineClient(ctx context.Context, cfg Config, network netconf.Network) (
 
 	return ethCl, nil
 }
+
+// enableSDKTelemetry enables prometheus based cosmos-sdk telemetry.
+func enableSDKTelemetry() error {
+	const farFuture = time.Hour * 24 * 365 * 10 // 10 years ~= infinity.
+
+	_, err := sdktelemetry.New(sdktelemetry.Config{
+		ServiceName:             "cosmos",
+		Enabled:                 true,
+		PrometheusRetentionTime: int64(farFuture.Seconds()), // Prometheus metrics never expire once created in-app.
```

Comment:
Don't we scrape them every 30s? Then we "ship" the metrics to Grafana - there we have "infinite" retention. I don't think we need metrics retention in the app as well.

Reply:
Some metrics never change ,like `up` and `git_commit_hash`